tools/circle_plus_gen/main.py

```diff
@@ -47,13 +45,13 @@ def inject_hparams(in_file, hparams_file, out_file=None) -> None:
         out_file = in_file
 
     tparams: TrainParam = TrainParam.from_json(hparams_file)
-    print("load training hyperparameters")
-    print(tparams.dump_as_json())
 
     circle_model: CirclePlus = CirclePlus.from_file(in_file)
     circle_model.set_train_param(tparams)
     print("succesfully add hyperparameters to the circle file")
 
+    print_training_hparameters(circle_model)
+
```

Comment:
This PR is to use `print_training_hparameters` here. 

Because I'll add a logic to print trainable operators inside of `print_training_hparameters`.